cmd/notify-webhook.go

```diff
@@ -48,33 +50,119 @@ type httpConn struct {
 	Endpoint string
 }
 
-// Lookup endpoint address by successfully dialing.
-func lookupEndpoint(u *url.URL) error {
-	dialer := &net.Dialer{
-		Timeout:   300 * time.Millisecond,
-		KeepAlive: 300 * time.Millisecond,
+// List of success status.
+var successStatus = []int{
+	http.StatusOK,
+	http.StatusAccepted,
+	http.StatusContinue,
+	http.StatusNoContent,
+	http.StatusPartialContent,
+}
+
+// isNetErrorRetryable - is network error retryable.
+func isNetErrorRetryable(err error) bool {
```

Comment:
This PR is supposed to ignore error like 'connection refused' for example when remote port is closed, can you add more comments to understand ?